### PR TITLE
Hotfix release to address minification issues

### PR DESCRIPTION
**Pull request by niik**
This PR is based off of the latest stable release (1.2.2) and cherry-picks c126c25bf1ed8803fdab56242a77f042c6bc2255 from #4871 which disables the built-in minification of webpack 4 (uglify-es) in favor of babel-minify.

We've chosen to only pull this particular fix in rather than releasing from master in order to minimize risk until we can give what's on master a more thorough run-through.

### CODE REVIEW

**Comment by niik** [comment]
1.2.3-beta1 and 1.2.3 has shipped off of this branch.

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,13 @@
 {
   "releases": {
+    "1.2.3": [
+      "[Fixed] No autocomplete when searching for co-authors - #4847",
+      "[Fixed] Error when checking out a PR from a fork - #4842"
+    ],
+    "1.2.3-beta1": [
```
Comment (say25) [inline changelog.json]: Maybe I am wrong but isn't this technically beta0 as it is the same as what is `1.2.3`?

Comment (shiftkey) [reply changelog.json]: @say25 yeah, it's in the [release process](https://github.com/desktop/desktop/blob/master/docs/process/releasing-updates.md#3-releasing) but oh well - version numbers are cheap.

**Comment by niik** [comment]
This has shipped and seems to function as intended, gonna merge it in so we don't forget

**Pull request merged**